berkeley-function-call-leaderboard/model_handler/oss_handler.py

```diff
@@ -34,78 +29,78 @@ def _format_prompt(prompt, function, test_category):
             functions += "\n" + str(function)
         return f"SYSTEM: {SYSTEM_PROMPT}\n{functions}\nUSER: {prompt}\nASSISTANT: "
 
-    @ray.remote(num_gpus=1)
-    @torch.inference_mode()
+    @staticmethod
     def _batch_generate(
-        question_jsons,
-        test_category,
+        test_question,
         model_path,
         temperature,
         max_tokens,
         top_p,
-        format_prompt_func,
-        index,
+        stop_token_ids=None,
+        max_model_len=None,
+        num_gpus=8,
```

Comment:
nit: `num_gpus` in `oss_handler` and `openfunctions_evaluation` are inconsistent.

Reply:
ah yes, good catch

Reply:
fixed